### ISSUE TITLE
About training code and data set

### ISSUE THREAD

**Issue by l1haodong** [JunHeum/BMBC#2]
Hi, nice work! 

Sorry to bother you, if it's convenient, would you consider uploading training codes and datasets later? 

Thank you sincerely！

**Comment by JunHeum** [comment]
Hi, l1haodong.

Sorry to late answer. we will upload train codes and datasets later.

If you want to use train codes as soon as possible, we recommend re-implement train codes.

In the paper, we describe the details of how to train.

Feel free to ask details of the training if you have any questions.

Thank you for your interest.

**Comment by 972461099** [comment]
> Hi, l1haodong.
> 
> Sorry to late answer. we will upload train codes and datasets later.
> 
> If you want to use train codes as soon as possible, we recommend re-implement train codes.
> 
> In the paper, we describe the details of how to train.
> 
> Feel free to ask details of the training if you have any questions.
> 
> Thank you for your interest.

Hi,I am try to re-implement train codes,but something I dont knew.
What is the training data set of Bilateral motion estimation and what is its label?

**Comment by JunHeum** [comment]
Hi, 972461099.

1. What is the training set of Bilateral motion estimation?

- As mentioned in the paper, we use the 'train set of  Vimeo90K' dataset to train the bilateral motion network.

2. What is its label?

- If what you said label was bilateral motion field ground truth, we use the intermediate frame of the triplet (consecutive three frames) as ground truth. 

- And we define the loss between the intermediate frame and the warped frame which is warped using predicted bilateral motion fields.

- Finally, the bilateral motion network learn to estimate accurate bilateral motion fields which decrease this loss.

Thank you for your interest!

**Comment by MaoYuxin25** [comment]
Hello, do you have any plans to open source the training code? I look forward to your answer.

**Comment by Yuqin-gif** [comment]
Hello, will you consider uploading your training code?
Thank you sincerely!
